requirements.txt

```diff
@@ -17,3 +17,4 @@ yuicompressor==2.4.8
 Flask-Admin==1.1.0
 Flask-Login==0.2.11
 WTForms==2.0.2
+Flask-Mail==0.9.1
```

Comment:
The only configuration it requires is the environmental variables:
app.config.update(dict(
    MAIL_SERVER='smtp.gmail.com',
    MAIL_PORT=465,
    MAIL_USE_TLS = False,
    MAIL_USE_SSL= True,
    MAIL_USERNAME = 'anywaytesttest@gmail.com',
    MAIL_PASSWORD = 'anyway987',
    MAIL_DEFAULT_SENDER = 'ANYWAY Team'
))

So you will set the real ones in the production environment?